server/src/main/java/io/crate/auth/HostBasedAuthentication.java

```diff
@@ -223,7 +231,7 @@ static boolean isValidConnection(String hbaConnectionMode, ConnectionProperties
             };
         }
 
-        private static long inetAddressToInt(InetAddress address) {
+        static long inetAddressToInt(InetAddress address) {
```

Comment:
`@VisibleForTesting` maybe or not important?

Reply:
Not that important, wouldn't hurt if this method is used within the same package. We could even promote it to public and move to `InetAddresses`